batch/src/main/scala/uk/gov/ons/addressindex/utils/Mappings.scala

```diff
@@ -12,14 +12,15 @@ object Mappings {
               "index": {
                   "similarity": {
                       "default": {
-                          "type": "classic"
+                          "type": "BM25"
                       }
                   }
               },
               "analysis": {
                   "filter": {
                       "address_synonym_filter": {
                           "type": "synonym",
+                          "lenient": "true",
```

Comment:
Were we getting format based errors?

Reply:
Without the lenient=true setting several of the synonyms in the mappings are rejected.

Reply:
I'm guessing the ones with numbers in.